crypto/fipsmodule/sha/internal.h

```diff
@@ -126,19 +108,18 @@ void sha512_block_data_order(uint64_t *state, const uint8_t *in,
 // SHA1_Init_from_state is a low-level function that initializes |sha| with a
```

Comment:
I think the repetition of the comments on the functions below can be reduced in the same way as with the macros. That would be more readable, I believe.